airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java

```diff
@@ -112,6 +122,61 @@ public JsonNode toJdbcConfig(final JsonNode config) {
     return Jsons.jsonNode(configBuilder.build());
   }
 
+  private List<String> getAdditionalParameters(final JsonNode config) {
+    final Map<String, String> customParameters = getCustomJdbcParameters(config);
+
+    if (useSSL(config)) {
+      return convertToJdbcStrings(customParameters, List.of(DEFAULT_JDBC_PARAMETERS, SSL_JDBC_PARAMETERS));
```

Comment:
You can use `MoreMappers#merge` to merge maps, and change the second parameter of `convertToJdbcStrings` to a map instead of a list of maps. This should simplify the `convertToJdbcStrings` method.